web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -2167,4 +2168,51 @@ describe("<BusinessFormationPaginator />", () => {
       page.fillText("Search business name", "Pizza Joint");
     };
   });
+
+  describe("remembers formation step", () => {
+    it("shows the first formation step with default formationStep", () => {
```

Comment:
I am not sure what this means... what is "default formationStep" 
I assume that means index 0 but:
- I would specify that in the test name to be clearer
- We should set index 0 explicitly on the user data in the "given" part here to make the cause-and-effect of the test clearer